src/org/usfirst/frc/team467/robot/Ramps.java

```diff
@@ -45,8 +45,8 @@ public static Ramps getInstance() {
 	}
 
 	public void deploy() {
-		if (state != State.START || DriverStation.getInstance().getMatchTime() > 30.0) {
-			// Only deploy from start configuration in the last 30 seconds
+		if (state != State.START || DriverStation.getInstance().getMatchTime() > 30.0 || !RobotMap.HAS_RAMPS) {
+			// Only deploy from start configuration in the last 30 seconds if you have ramps
```

Comment:
Ok, you've gone too complex. Break it down:
`if (state != State.START) {`
`    return;`
`}`
``
`if (DriverStation.getInstance().getMatchTime > 30.0) {`
`    return;`
`}`

etc.